docs/configuration.md

```diff
@@ -75,6 +75,7 @@ Unless you need to set a non-default value, it is recommended to only populate o
 | `HBAR_RATE_LIMIT_BASIC`                     | "1120000000"               | Individual limit (in tinybars) for spending plans with a BASIC tier. Defaults to 11.2 HBARs.                                                                                                                                                                                                           |
 | `HBAR_RATE_LIMIT_EXTENDED`                  | "3200000000"               | Individual limit (in tinybars) for spending plans with a EXTENDED tier. Defaults to 32 HBARs.                                                                                                                                                                                                          |
 | `HBAR_RATE_LIMIT_PRIVILEGED`                | "8000000000"               | Individual limit (in tinybars) for spending plans with a PRIVILEGED tier. Defaults to 80 HBARs.                                                                                                                                                                                                        |
+| `HBAR_SPENDING_PLANS_CONFIG_JSON`           | "" | Spending plans can be loaded from environment varaibles in JSON format, based on the spendingPlansConfig.example.json file.                                    |
```

Comment:
**nit:** Format the column widths of this row to be consistent with other rows in the table.

Reply:
Fixed.  Removed line when renamed to `HBAR_SPENDING_PLANS_CONFIG`